post_installation_configuration/cluster-capabilities.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="cluster-capabilities"]
+= Cluster capabilities
+include::_attributes/common-attributes.adoc[]
+:context: cluster-capabilities
+
+toc::[]
+
+The cluster capabilities provide an option for cluster administrators to select or deselect one or more optional components at any time post installation.
```

Comment:
User can select capability during installation too. So the above statement is incorrect. We should change it to 
```
The cluster capabilities provide an option for cluster administrators to select or deselect one or more optional components prior to installation and ability to enable a capability anytime post installation.
```